### ISSUE TITLE
does the DiffAugment not be used in train_vqgan?

### ISSUE THREAD

**Issue by YilanWang** [samb-t/unleashing-transformers#17]
thx for this great work, I find it seems that the DiffAugment is not used in train_vqgan, neither train LSUN nor ffhq?

**Comment by samb-t** [comment]
When training all models we used DiffAug enabling it by passing in the flag `--diff_aug` to `train_vqgan.py`. We must have forgotten to change this to be the default in the config files when cleaning up the code. When I get time I'll update this properly but for now just use this flag (I've updated the ReadMe with this as well). Thank you!

**Comment by YilanWang** [comment]
thx, And I have another question,before the optimizer learns ganloss(step<disc_start_step), the loss curve will decrease very smoothly, but after adding gan loss, the loss curve will be very sharp, and the recon quality will decrease some steps. Is it right?

**Comment by samb-t** [comment]
Yes that's right, we observe the same in all cases. This is because the blurry reconstructions are easily identifiable as fake by the discriminator so the reconstructions quickly change to trick the discriminator which typically results in worse reconstructions. After not too long it will recover and the reconstructions will look much better, so this is nothing to worry about. The loss curve will likely continue to be very sharp as the adversarial loss dominates in magnitude, but this is fine. While the default hyperparameters were fine for LSUN and FFHQ in this paper, we have subsequently found that on some other smaller datasets, if the reconstruction quality is not good enough then reducing the discriminator weight and annealing it in can help.

**Comment by YilanWang** [comment]
many thanks for your reply！I close this issue.
